packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -43,7 +43,7 @@ export const microphoneButtonSelector: MicrophoneButtonSelector = reselect.creat
   (callExists, isMuted, deviceManager) => {
     const permission = deviceManager.deviceAccess ? deviceManager.deviceAccess.audio : true;
     return {
-      disabled: !callExists || !permission,
+      disabled: !callExists || !permission || !deviceManager.microphones.length,
```

Comment:
I am assuming the drop down still works allowing users to change their speakers?